python/packages/streamsx/rest_primitives.py

```diff
@@ -50,17 +63,76 @@ def get_instance(self):
     def get_job(self):
         return Job(self.rest_client.make_request(self.job), self.rest_client)
 
+    def stop_data_fetch(self):
+        self._stop.set()
+
+    def start_data_fetch(self):
+        self._stop.clear()
+        t = threading.Thread(target=self)
```

Comment:
So `View` is a thread and we create a thread here? Just unclear what is going on.